draft-ietf-tls-esni.md

```diff
@@ -1829,6 +1829,35 @@ registration policy ({{!RFC8126, Section 4.6}}).
 The registration policy for for the "ECHConfig Extension Type" registry
 is Specification Required {{!RFC8126}}.
 
+This document defines several Reserved values for ECH configuration extensions.
+These can be used by clients and servers to "grease" the contents of the
```

Comment:
Clients don't actually send these, so I think this should be:

```suggestion
These can be used by servers to "grease" the contents of the
```

Reply:
Actually, it's the ECHConfig generator not the TLS server that does this.

Reply:
Well, unless we're expecting to GREASE retry_configs? I'd not thought about that.